scripts/gen-s-parser.py

```diff
@@ -708,6 +708,10 @@ def instruction_parser():
 
     printer.print_line("auto op = *keyword;")
     printer.print_line("char buf[{}] = {{}};".format(inst_length + 1))
+    printer.print_line("// Ensure we do not copy more than the buffer can hold")
+    printer.print_line("if (op.size() >= sizeof(buf)) {")
+    printer.print_line("	goto parse_error;")
```

Comment:
```suggestion
    printer.print_line("  goto parse_error;")
```
(spaces instead of tabs)